aspnet/tutorials/your-first-mac-aspnet.rst

```diff
@@ -1,53 +1,88 @@
 
 Scaffolding Applications Using Yeoman
 -------------------------------------
 
-Follow the instruction in :doc:`/client-side/yeoman` to create an ASP.NET Core project.
+We will be using ``yo aspnet`` to generate the **Web Application Basic** template, you may follow the full instructions in :doc:`/client-side/yeoman` to create an ASP.NET Core project which show an **Empty Web** for reference.
+
+Install the necessary yeoman generators and bower (for client side assets) using npm.
+
+.. code-block:: console
+
+    npm install -g yo generator-aspnet bower
+
+Run the ASP.NET Core generator
+
+.. code-block:: console
+
+    yo aspnet
+
+* Select **Web Application Basic [without Membership and Authorization]** and tap Enter
+* Select Bootstrap (3.3.6) as the UI framework and tap Enter
+* Use "MyFirstApp" for the app name and tap Enter
+
+When the generator completes scaffolding the files, it will instruct you to restore, build, and run the application.
+
+.. code-block:: console
+
+    Your project is now created, you can use the following commands to get going
+        cd "MyFirstApp"
+        dotnet restore
+        dotnet build (optional, build will also happen with it's run)
+        dotnet run
+
+.. note:: Mac keyboard symbols - ⌘ : Command, ⌃ : Ctrl, ⇧ : Shift, ⌥ : Option
 
 Developing ASP.NET Core Applications on a Mac With Visual Studio Code
 ---------------------------------------------------------------------
 
 - Start **Visual Studio Code**
 
-.. image:: your-first-mac-aspnet/_static/vscode-welcome.png
-
-- Tap **File > Open** and navigate to your Empty ASP.NET Core app
+- Tap **File > Open** and navigate to your ASP.NET Core app
 
 .. image:: your-first-mac-aspnet/_static/file-open.png
 
-From a Terminal / bash prompt, run ``dotnet restore`` to restore the project's dependencies. Alternately, you can enter ``command shift p`` in Visual Studio Code and then type ``dot`` as shown:
+When the application is opened, Visual Studio Code will prompt to restore the needed project dependencies as well as add build and debug dependencies. 
+
+.. image:: your-first-mac-aspnet/_static/dependencies-restore.png 
+    :scale: 100%
+
+Tap "Yes" to add the build and debug assets.
 
-.. image:: your-first-mac-aspnet/_static/dotnet-restore.png
+.. image:: your-first-mac-aspnet/_static/debug-items-added.png 
+    :scale: 100%
 
-You can run commands directly from within Visual Studio Code, including ``dotnet restore`` and any tools referenced in the *project.json* file, as well as custom tasks defined in *.vscode/tasks.json*.
+Tap "Restore" to restore the project dependencies. Alternately, you can enter ``⌘⇧P`` in Visual Studio Code and then type ``dot`` as shown:
```

Comment:
While the Mac UI will use these symbols for indicating keyboard shortcuts, it doesn't look like Apple uses these symbols in their docs: https://support.apple.com/en-us/HT201236